.github/workflows/ci-pipeline.yml

```diff
@@ -0,0 +1,63 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request for master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: 'Install OS Packages'
+        run: |
+          sudo apt update
+          sudo apt --fix-broken install -y
+          sudo apt install python3-pip -y
+          pip3 install setuptools
+          pip3 install pylint coverage
+        
+
+      - name: 'Install Docker'
```

Comment:
I think Docker is already installed on `ubuntu-latest`. You might not need this, unless you need the latest version?